### PR TITLE
Fixes #74 -- make it more obvious to refresh than to uncerify

### LINKED ISSUE #74: Start/Doc Comparison Page: Highlight the ability to refresh the page
Many participants hard a hard time finding the "refresh" link after they had corrected missing file dates in VBMS. 

Suggestion: Consider adding a "Refresh" button down next to the "This case cannot be certified" button and/or auto-refreshing the page.

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

**Comment by alex** [comment]
screen shot 2015-10-28 at 3 17 13 pm

File: app/views/web/start.html.erb [hunk 1]
```diff
@@ -73,11 +73,15 @@
...
-
-          <%= form_tag(action: :error) do %>
-            <input type="submit" value="This case cannot be certified" class="btn btn-primary"/>
+          <strong>If you can't find the missing file in VBMS</strong> click the link below to report this as un-certifiable.
+        </p>
+        <p>
+          <%= form_tag({action: :error}, {class: "pull-right"}) do %>
+            <input type="submit" value="This case cannot be certified" class="btn btn-link"/>
           <% end %>
+          <a href="#" class="btn btn-primary" onclick="location.reload()">
```
Comment (paultag) [inline app/views/web/start.html.erb]: Is there a reason to have this be a `<a>` rather than a `<button>`?

**Comment by paultag** [comment]
Looks good to me! Bouncing to @KHarshawat to ACK the design bits!

**Comment by alex** [comment]
screen shot 2015-10-28 at 4 18 42 pm

**Pull request merged**